src/SQLite.jl

```diff
@@ -155,13 +182,14 @@ function sqlserialize(x)
     serialize(t,s)
     return takebuf_array(t)
 end
-bind(stmt::SQLiteStmt,i::Int,val) = bind(stmt,i,sqlserialize(val))
+"bind `val` to the parameter at index `i`"
```

Comment:
Maybe make it clearer that this is a fallback method and that it will be serialized as a `BLOB`.